src/customize-pop/online-accounts.md

```diff
@@ -0,0 +1,44 @@
+# Online Accounts
+
+Settings to connect online accounts are located in `Settings` ➞ `Online Accounts`. Online Accounts settings provide single sign-on for online providers. Connected accounts can directly access core services like Mail, Calendar, Contacts, Maps, Photos, Files, and Music, without requiring sign-on through a web browser.
+
+---
+
+## Required Data Access
+
+This chart is also found on the [GNOME wiki](https://wiki.gnome.org/Projects/GnomeOnlineAccounts/Providers).
+
+| Provider | Mail | Calendar | Contacts | Maps | Photos | Files | Ticketing | Printers | Music |
+|----------|------|----------|----------|------|--------|-------|-----------|----------|-------|
+| Facebook |      |          |          | Yes  | Yes    |       |           |          |       |
+| Flickr   |      |          |          |      | Yes    |       |           |          |       |
+| Foursquare |    |          |          | Yes  |        |       |           |          |       |
+| Google   | Yes  | Yes      | Yes      |      | Yes    | Yes   |           | Yes      |       |
+| Microsoft| Yes  |          |          |      |        |       |           |          |       |
+| Microsoft Exchange | Yes | Yes | Yes  |      |        |       |           |          |       |
+| Nextcloud |     | Yes      | Yes      |      |        | Yes   |           |          |       |
+| IMAP & STMP | Yes |        |          |      |        |       |           |          |       |
+| Kerberos |      |          |          |      |        |       | Yes       |          |       |
+| Last.fm  |      |          |          |      |        |       |           |          | Yes   |
+
+## Connecting an Online Account
+
+1. The registration process varies depending on the services of the provider.
```

Comment:
This is not a step.

Reply:
Good catch, I've updated this intro and moved this image to the top of the section.